.test-env

```diff
@@ -1,11 +1,11 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="pre-consensus"
+SDK_TESTING_BRANCH="verbose-harness"
 SDK_TESTING_HARNESS="test-harness"
 
 INSTALL_ONLY=0
 
-VERBOSE_HARNESS=0
+VERBOSE_HARNESS=1
```

Comment:
Once the `sandbox` and `sdk-testing` PR's are merged, this is the only change that is needed in the other SDK's

Reply:
LGTM - do we want to target the `develop` branch instead or roll in this change with this branch?

Reply:
But if you prefer to roll it into `txn-deltas-api` that works as well.

Reply:
I think targeting `develop` is fine since indexer is building on sandbox again!